src/plugins/run/annotations.ts

```diff
@@ -189,10 +189,29 @@ function processNode(node: MDAST.Content, annotations?: object): MDAST.Content {
       annotations,
       data: {
         ...(node.data || {}),
-        hProperties: annotations,
+        hProperties: normalizeAnnotationsForHast(annotations),
       },
     };
   }
 
   return node;
 }
+
+// micromark ecosystem passes data through html, and then to react
+// HTML does not allow for boolean properties, so here we stringify boolean values so that they can pass through
```

Comment:
> HTML does not allow for boolean properties

How so?
There are boolean attributes.
https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#boolean-attributes

Reply:
I meant no boolean attributes like `disabled=true`. The html boolean attributes basically translate to `disabled=""`, and as far as I'm aware there is no way to pass false, so really it's a "true" attribute (not boolean). Currently in `markdown-viewer` it's checking if these boolean attributes do or do not equal `""`, which just seemed more confusing that checking against `"true"` or `"false"`. But tbh I don't mind a whole lot either way, just thought that explicit "true" / "false" was more clear than "".

Reply:
> and as far as I'm aware there is no way to pass false, so really it's a "true" attribute (not boolean)

yeah, as the spec states:

> The presence of a boolean attribute on an element represents the true value, and the absence of the attribute represents the false value.

Reply:
> But tbh I don't mind a whole lot either way, just thought that explicit "true" / "false" was more clear than "".

It's actually less clear IMHO. We should try to stick to HTML, I think.